### DISCUSSION TITLE
How to increase the file size limit for import function

### DISCUSSION THREAD

**Discussion by yu929** [outline/outline#5152] [Self Hosting] [answered]
When I use the import function, the interface prompts that the file is too large. The AWS_S3_UPLOAD_MAX_SIZE in the env file has been set to 26214400, and the attachment function in the document works without any problem. However, the import function is still restricted.
![QQ图片20230404142804](https://user-images.githubusercontent.com/36434225/229707867-bd7170c5-b7c2-4627-84ab-63637881e16b.png)

**Comment by tommoor** [comment, marked as answer]
There is also a `MAXIMUM_IMPORT_SIZE` env variable that you can set, it allows a larger file size for imports over standard document attachments.
